README.md

```diff
@@ -30,79 +30,59 @@ yarn install rxjs-audio
 
 ## Usage
 
-In order to `play` the audio files/urls. You need to create first `AudioStream` Object via `rxjsAudioService.create` method, as shown below:
+### Initialize the stream
+
+Before using, you need to create an object of the Stream class. You'll need to pick a class depening on your use-case: 
+- `AudioStream` is the base-class similar to the html5 audio element where you can load a track. 
+- `PlaylistAudioStream` is like the name says more suitable for playing playlists, in the options you'll be able to add an array of tracks
 
-### Creating a Stream
 ```ts
-import { RxJSAudioService } from 'rxjs-audio';
+import { AudioStream } from 'rxjs-audio-stream';
```

Comment:
Can you please rename it back to the original name of the package? or there is a specific reason you did it?

Reply:
My bad, will update this